src/raylib/core/Sound.hpp

```diff
@@ -0,0 +1,62 @@
+/*
+** EPITECH PROJECT, 2022
+** Bomberman-Global-Offensive
+** File description:
+** Music
+*/
+
+#ifndef RAYLIB_CORE_MUSIC_HPP_
+#define RAYLIB_CORE_MUSIC_HPP_
+
+extern "C"
+{
+#include <raylib.h>
+}
+
+#include <filesystem>
+#include <string>
+
+/// raylib namespace
+namespace raylib
+{
+    namespace core
+    { /// Core namespace (inside of raylib)
+        class Sound {
+          public:
+            /// Consutructor of Sound wich create the sounds
+            ///
+            /// @param filename Sound path
+            Sound(const std::filesystem::path &fileName);
+            ~Sound();
+
+            /// Check if sound is playing now
+            ///
+            /// @retval true if the sound is currently playing
+            /// @retval false if the sound is not currently playing
+            bool isSoundPlaying();
```

Comment:
The Class is Sound, you don't have need to say "isSoundPlaying" you can use isPlaying. We know that function is use on Sound Object.